packages/core/src/shipping/strategies/paypal-commerce/paypal-commerce-fastlane-shipping-strategy.ts

```diff
@@ -75,7 +71,7 @@ export default class PayPalCommerceFastlaneShippingStrategy implements ShippingS
 
                 this._isFastlaneEnabled = !!paymentMethod.initializationData.isFastlaneEnabled;
 
-                await this._initializePayPalSdk(methodId, styles);
+                await this._initializePayPalSdk(methodId, paypalcommercefastlane?.styles);
```

Comment:
Same question as before.

Reply:
Same answer as above. It should be called only for Fastlane related methods